resources/leiningen/new/kraken_http_api/service.clj

```diff
@@ -8,7 +8,10 @@
             [pedestal-toolbox.params :refer :all]
             [pedestal-toolbox.content-negotiation :refer :all]
             [kehaar.core :as k]
-            [clojure.core.async :refer [chan go alt! timeout]]))
+            [clojure.core.async :refer [go alt! timeout]]
+            [bifrost.core :as bifrost]
+            [bifrost.interceptors :as bifrost.i]
```

Comment:
Hmm, I would worry that `interceptors` isn't very revealing in a pedestal app. If I saw that prefix I'd have to go look up what it is every time because "intereceptors" are everywhere! :)

Whereas `bifrost.i` is what we're using in other http-api components (not that that means we *have* to keep it) and tells us that whatever this is prefixing, it comes from the bifrost library.